cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+
+### Further Stipulations Related to OCI conda Chennal Repositories and Tags
+
+- If either of the `<OCI-encoded package name>` or the OCI tag for the conda package are hashed, then both MUST be hashed.
+- If an OCI registry implementation errors due to length when dealing with the full OCI conda URL for the package, then
+  that package cannot be stored in that registry and an error MUST be raised.
+- The encoding rules defined above are solely for internal storage of conda packages within an OCI registry. Outside of
+  dealing with the OCI registry itself, the original forms of the channel, subdir, label, name, version and build string MUST be used.
+
+### Allowed Blob mediaTypes
+
+We define the following custom media types that MUST be used for the storage of conda repodata, run exports, channel data,
+and packages in an OCI registry:
+
+| Blob type            | Content type                | mediaType                                      |
+|----------------------|-----------------------------|------------------------------------------------|
+| conda package v1     | .tar.bz2 package            | application/vnd.conda.package.v1               |
+| conda package v2     | .conda package              | application/vnd.conda.package.v2               |
+| package info         | `info` folder as gzip       | application/vnd.conda.info.v1.tar+gzip         |
+| package index        | `info/index.json` file      | application/vnd.conda.info.index.v1+json       |
+| repodata.json        | `repodata.json` file        | application/vnd.conda.repodata.v1+json         |
+| repodata.json.zst    | `repodata.json.zst` file    | application/vnd.conda.repodata.v1+json+zst     |
+| repodata.json.gz     | `repodata.json.gz` file     | application/vnd.conda.repodata.v1+json+gzip    |
+| repodata.json.bz2    | `repodata.json.bz2` file    | application/vnd.conda.repodata.v1+json+bz2     |
+| channeldata.json     | `channeldata.json` file     | application/vnd.conda.channeldata.v1+json      |
+| run_exports.json     | `run_exports.json` file     | application/vnd.conda.run_exports.v1+json      |
+| run_exports.json.zst | `run_exports.json.zst` file | application/vnd.conda.run_exports.v1+json+zst  |
+| run_exports.json.gz  | `run_exports.json.gz` file  | application/vnd.conda.run_exports.v1+json+gzip |
+| run_exports.json.bz2 | `run_exports.json.bz2` file | application/vnd.conda.run_exports.v1+json+bz2  |
+| repodata_from_packages.json | `repodata_from_packages.json` file | application/vnd.conda.repodata_from_packages.v1+json |
+| repodata_from_packages.json.zst | `repodata_from_packages.json.zst` file | application/vnd.conda.repodata_from_packages.v1+json+zst |
+| repodata_from_packages.json.gz | `repodata_from_packages.json.gz` file | application/vnd.conda.repodata_from_packages.v1+json+gzip |
+| repodata_from_packages.json.bz2 | `repodata_from_packages.json.bz2` file | application/vnd.conda.repodata_from_packages.v1+json+bz2 |
```

Comment:
These follow the same schema as other repodatas, but are unpatched. Do we need a separate media type? What about `current_repodata`?

Reply:
Ah good point. I'll condense to a single repodata schema and add current repodata. Unfortunately current repodata starts with a "c" and so conflicts with a package named _repdata.json. We'll have to resolve that conflict somehow.

Reply:
Argh, do we need to prefix something to all `repodata`s then? Because in that case we should prepend the same prefix to all non-package information for consistency. Something like `m`. So `repodata.json` would be stored under `mrepodata.json`, and `patch_instructions.json` would be under `mpatch_instructions.json`?

Reply:
Sure, that can work. It would just be odd because in a way we are limiting `repodata.json`-like files to _not_ start with `h` or `c`, and at the same time allowing all the other letters implicitly, while also requiring some copies to start with `r`? So I'd rather have a clear and simple rule for non-package files, so they all must start with the same prefix.

Reply:
I see what you mean. That is indeed better so let's go with `m` for `metadata`. I will adjust the CEP.